services/workflows-service/src/workflow/workflow-controller.module.ts

```diff
@@ -0,0 +1,11 @@
+import { Module } from '@nestjs/common';
```

Comment:
`services/workflows-service/src/workflow/workflow-controller.module.ts` ? dosent make sense

Reply:
Separated wf-service & controller to different modules to simplify re-using of wf-service logic in different modules. Instead of re-import everything in order to make wf-service run now we can import just single workflow module which exports service.